assets/src/scss/layout/navbar/languages.scss

```diff
@@ -12,11 +12,28 @@
     display: inline-block;
     line-height: $menu-height;
 
+    &::after {
+      content: "";
+      display: inline-block;
+      width: 10px;
+      height: 10px;
+      margin-inline-start: $sp-x;
+      mask: url("../../images/chevron.svg") 0 0/10px 10px;
+      transform: rotate(90deg);
+      transition: transform 150ms linear;
+      background-color: $white;
```

Comment:
This shouldn't be white when the `Light` navbar is selected:
<img width="177" alt="Screenshot 2022-05-11 at 16 01 42" src="https://user-images.githubusercontent.com/6949075/167868520-e14038a0-1809-439d-ad5e-a0813dc1b9f8.png">